### DISCUSSION TITLE
Hide The Top and Bottom Navigation Bars During a Workout

### DISCUSSION THREAD

**Discussion by feraask** [astashov/liftosaur#138] [Ideas]
The top and bottom navigation bars can take up a lot of screen space during a workout, many apps simply hide these when scrolling down a list and will show them again if you scroll up, would be nice if Liftosaur implemented something similar:
![image](https://github.com/user-attachments/assets/ec0794cd-5389-406d-beb5-37408c8b4d73)

Alternatively, a full-screen toggle button or something could also work to hide/show them on demand if we don't want it tied to scrolling.

**Comment by astashov** [comment]
Curious - what's the device? It does look pretty condensed.

**Reply by feraask** [reply]
The image in the post was just a screenshot taken running the web-app on my Windows 10 PC.

I have a home gym with a large display that I hook my laptop up to and run Liftosaur as web-app to track my workouts. I like to have Liftosaur always visible throughout my workout so I will zoom it a lot so I can read the text for sets/reps/weight and exercise descriptions easily from afar.

Navigation bars end up taking a lot of space this way. Would be really nice if they could be hidden easily.

On phone you can run into something similar if you set the screen zoom higher for bigger touch targets.

Here's how it looks on my Samsung Galaxy Z Flip 5:
![image](https://github.com/user-attachments/assets/ffeb0810-c4a4-4bf8-a49d-14e13f3ff2dc)
